pom.xml

```diff
@@ -26,6 +26,7 @@
   </parent>
   <groupId>org.apache.spark</groupId>
   <artifactId>spark-parent_2.11</artifactId>
+  <!-- When updating version make sure to update ./python/pyspark/version.py as well -->
```

Comment:
I rather not have a regex do that, that just means it will break and we won't notice it. (joking: One of the ways to summon cuthulu is to use a regex to parse HTML - I think using a regex to parse the version number from XML won't quite give us cuthulu but I'd rather not risk it).

If there is a consensus that updating the version string in two places is going to be error prone I can add some code for this but I think most solutions to this end up being as brittle as the human process (and less likely to be noticed).